docs/zh_cn/train.md

```diff
@@ -59,7 +59,7 @@ GPUS=16 ./tools/slurm_train.sh dev pspr50 configs/pspnet/pspnet_r50-d8_512x1024_
 您可以查看 [slurm_train.sh](../tools/slurm_train.sh) 以熟悉全部的参数与环境变量。
 
 如果您多个机器已经有以太网连接， 您可以参考 PyTorch
-[launch utility](https://pytorch.org/docs/stable/distributed_deprecated.html#launch-utility) 。
+[launch utility](https://pytorch.org/docs/en/stable/distributed_deprecated.html#launch-utility) 。
```

Comment:
This is a link to pytorch.org :)

Reply:
Both `https://pytorch.org/docs/en/stable/distributed_deprecated.html#launch-utility` and `https://pytorch.org/docs/stable/distributed_deprecated.html#launch-utility` are dead links.

Reply:
I just copied related url from [mmdet](https://github.com/open-mmlab/mmdetection/blob/master/docs/1_exist_data_model.md), it is `https://pytorch.org/docs/stable/distributed.html#launch-utility`.